devel/python/CMakeLists.txt

```diff
@@ -1,28 +1,40 @@
 include(cmake/find_python_module.cmake)
-include(cmake/python_module_versions.cmake) # finds version
+include(cmake/pycmake.cmake) # finds version
```

Comment:
It seems varieties of `CamelCAPS` is all the rage when it comes to [cmake modules](https://cmake.org/cmake/help/v3.0/manual/cmake-modules.7.html) - so maybe this should be `PyCMake`?